server/admin_management/api/app/routers/router_application_admin.py

```diff
@@ -144,7 +144,7 @@ def delete_application_admin(
     status_code=200,
     dependencies=[Depends(authorize_by_fam_admin)],
 )
-def get_application_admin_by_applicationid(
+def get_application_admin_by_application_id(
```

Comment:
rename this method just for naming consistency, forgot to change it in the last pr

Reply:
Nice!